src/Security/VoterV2/ProVoter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Security\VoterV2;
+
+use App\Entity\MembreCentre;
+use App\Entity\User;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+class ProVoter extends Voter
+{
+    public const MANAGE = 'MANAGE_PRO';
+
+    protected function supports(string $attribute, $subject): bool
+    {
+        return self::MANAGE == $attribute;
+    }
+
+    protected function voteOnAttribute(string $attribute, $subject, TokenInterface $token): bool
+    {
+        $user = $token->getUser();
+        if (!$user instanceof User) {
+            return false;
+        }
+
+        return $user->hasDroit(MembreCentre::TYPEDROIT_GESTION_MEMBRES);
```

Comment:
First, I'm not sure that hasDroit() should exist in User, as Membre is the only child class which can have this type of permission.
Moreover, having the right permission isn't enough, we need to check that the pro has accepted affiliation with the center on which he has these rights